### PR TITLE
Fix/profile

### PR DESCRIPTION
**Pull request by mneokvsenorm**

### CODE REVIEW

File: src/components/ProfileEdit/index.js [hunk 1]
```diff
@@ -61,5 +72,4 @@ const ProfileEdit = ({ handleSubmit, classes, onSave, toggle }) => (
 
 export default reduxForm({
```
Comment (ZelikSV) [inline src/components/ProfileEdit/index.js]: You should use recompose for this

File: src/reducers/userReducer/user.js [hunk 2]
```diff
@@ -1,28 +1,35 @@
...
     case FETCH_USER_SUCCESS:
-      return state.merge({
-        profile: fromJS(payload),
-      });
+      return {
+        ...state,
+        profile: payload,
+      };
     case SAVE_PROFILE:
       console.log(payload);
```
Comment (ZelikSV) [inline src/reducers/userReducer/user.js]: Remove console.log

File: src/components/ProfileEdit/index.js [hunk 3]
```diff
@@ -25,25 +25,36 @@ const ProfileEdit = ({ handleSubmit, classes, onSave, toggle }) => (
...
             component={Input}
             label="Name"
             Icon={AccountIcon}
             className={classes.input}
+            validate={validateTextEmpty}
+          />
+          <Field
+            name="lastName"
+            component={Input}
+            label="Name"
```
Comment (ZelikSV) [inline src/components/ProfileEdit/index.js]: Change name for labels. They must be not same

File: src/api/getProfile/index.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+import axios from 'axios';
+
+// eslint-disable-next-line import/prefer-default-export
```
Comment (AndrZakhara) [inline src/api/getProfile/index.js]: Delete this line

**Pull request merged**